spec/horoshop_spec.rb

```diff
@@ -1,33 +1,29 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
-require 'faraday'
 
 describe Horoshop do
+  subject(:horoshop) { described_class.new(params) }
```

Comment:
`subject { described_class.new(params) }`
do not use variables names for SUBJECT !!!!

`expect(subject).to something`